### ISSUE TITLE
constructor-super rule only detects calls on their own line

### ISSUE THREAD

**Issue by nathanielyoon** [denoland/deno_lint#1490]
### Lint Name

constructor-super

### Code Snippet

```ts
class A {}
export class B extends A {
  constructor() {
    super();
  }
}
export class C extends A {
  constructor() {
    super(), 0;
  }
}
export class D extends A {
  constructor() {
    0, super();
  }
}
```

### Expected Result

No lint error, `super()` is called unconditionally in all 3 cases.

### Actual Result

```
tmp $ deno lint --compact example.ts 
file:///tmp/example.ts: line 8, col 3 - Constructors of derived classes must call super() (constructor-super)
file:///tmp/example.ts: line 13, col 3 - Constructors of derived classes must call super() (constructor-super)
Found 2 problems
Checked 1 file
```

### Additional Info

From [issue on deno repo](https://github.com/denoland/deno/issues/33975). May be related to #1272.

### Version

deno 2.7.13 (stable, release, x86_64-unknown-linux-gnu)
v8 14.7.173.20-rusty
typescript 5.9.2
